Makefile

```diff
@@ -78,6 +78,8 @@ else
  detected_OS := $(strip $(shell uname))
 endif
 
+QMAKE ?= $(shell which qmake)
```

Comment:
Is it possible to also configure the cmake variables for DOtherSide and StatusQ to use the same qt?